### PR TITLE
 tables created and migrated

### PR DESCRIPTION
**Pull request by AspiringDev51**
Purpose: Created the Tables for StructureFire and StructureFireLink to store data from  scraping  these pdf files https://www.okc.gov/home/showpublisheddocument/37021/638276036304930000

how to test:  
bundle exec  rspec structure_fire_spec.rb
bundle exec  rspec structure_fire_link_spec.rb

### CODE REVIEW

File: spec/models/structure_fire_link_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe StructureFireLink, type: :model do
```
Comment (holdenmitchell) [inline spec/models/structure_fire_link_spec.rb:3]: Update spec based on the model changes

**Review by holdenmitchell — CHANGES_REQUESTED** [review]
A couple of small tweaks 🔧

File: db/schema.rb [hunk 2]
```diff
@@ -583,6 +583,32 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "structure_fire_links", force: :cascade do |t|
+    t.string "url"
+    t.date "date"
```
Comment (AspiringDev51) [inline db/schema.rb:588]: pdf_date_on

**Review by holdenmitchell — APPROVED** [review]
Looks good 🚒 🔥

**Pull request merged**